02_bubble/sarahdavy1983/main.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"fmt"
+)
+
+func bubbleSort(s []int) []int {
+	sliceCopy := append(s[:0:0], s...)
+	n := len(s)
+	swapped := true
+	for swapped {
```

Comment:
I like the use of this; although to simplify we could use 
`for swapped := true; swapped;{ `if we wanted to condense a little more